src/printutils.cc

```diff
@@ -31,6 +37,11 @@ void set_output_handler(OutputHandlerFunc *newhandler, void *userdata)
 	outputhandler_data = userdata;
 }
 
+void set_output_handler2(OutputHandlerFunc2 *newhandler, void *userdata)
```

Comment:
I wonder if it's better to combine the 2 setters, we don't expect different `userdata` (different MainWindow) in any case I think. So one pointer should be enough.

Reply:
Done :)